kong/upstream_node_health_service.go

```diff
@@ -0,0 +1,61 @@
+package kong
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+)
+
+// MTLSAuthService handles Upstream Node Healths in Kong.
```

Comment:
Looks like a typo here `MTLSAuthService -> UpstreamNodeHealthService`

Reply:
Good catch! Fixed the typo.